extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -521,6 +521,11 @@ export class AmpSlideScroll extends BaseSlides {
    * @return {number} a number representing the next slide index.
    */
   getNextSlideIndex_(currentScrollLeft) {
+    // Addresses race where slideWidth is 0, due to being closed
```

Comment:
I'm not sure I understand what it means for the carousel to be closed?

Reply:
Within the lightbox gallery sense, you open the carousel by clicking on the lightbox image and then close it by clicking the image in the carousel.

Reply:
Oh I see. In that case perhaps we can update this to `hidden`?